### PR TITLE
Fixed a Arguement Error

### PR DESCRIPTION
**Pull request by CyberiumShadow**
func_name_matching had off and always together.

### CODE REVIEW

File: packages/eslint-config-airbnb-base/rules/style.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ module.exports = {
     // requires function names to match the name of the variable or property to which they are
     // assigned
     // http://eslint.org/docs/rules/func-name-matching
-    'func-name-matching': ['off', 'always', {
```
Comment (ljharb) [inline packages/eslint-config-airbnb-base/rules/style.js]: actually, this is correct. the string option is optional, but our configs prefer to be explicit.

**Comment by ljharb** [comment]
If you're running into an eslint error, it's because you have to upgrade your version of eslint.

**Comment by CyberiumShadow** [comment]
@ljharb What would need to be updated then?

**Comment by ljharb** [comment]
I'd try restarting atom - it's got a bunch of bugs around changing node_modules while remaining open.

**Comment by CyberiumShadow** [comment]
Turns out that installing Node 6.9.1 wiped my global node modules.
All good now, I also had Global ESLint enabled for my linter.

Sorry for the trouble!

**Comment by ljharb** [comment]
Glad to hear it - you definitely want everything installed locally.

**Pull request closed without merging**